opensrp-reveal/src/test/java/org/smartregister/reveal/activity/ChildProfileActivityTest.java

```diff
@@ -0,0 +1,203 @@
+package org.smartregister.reveal.activity;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.view.View;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import org.codehaus.plexus.util.StringUtils;
+import org.hamcrest.CoreMatchers;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.android.controller.ActivityController;
+import org.robolectric.annotation.Config;
+import org.robolectric.shadows.ShadowActivity;
+import org.robolectric.shadows.ShadowIntent;
+import org.robolectric.util.ReflectionHelpers;
+import org.smartregister.Context;
+import org.smartregister.CoreLibrary;
+import org.smartregister.family.activity.FamilyWizardFormActivity;
+import org.smartregister.family.util.JsonFormUtils;
+import org.smartregister.reveal.TestRevealApplication;
+import org.smartregister.reveal.contract.ChildProfileContract;
+import org.smartregister.reveal.model.Child;
+import org.smartregister.reveal.view.ChildProfileActivity;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+import static org.robolectric.Shadows.shadowOf;
+
+/**
+ * @author ronald
+ */
+
+@RunWith(RobolectricTestRunner.class)
+@Config(application = TestRevealApplication.class, sdk = 22)
+public class ChildProfileActivityTest {
+
+    @Rule
+    public MockitoRule rule = MockitoJUnit.rule();
+
+    private ChildProfileActivity activity;
+    private String childBaseID = "childBaseID";
+    private ActivityController<ChildProfileActivity> controller;
+
+    @Mock
+    private ChildProfileContract.Presenter presenter;
+
+    @Before
+    public void setUp() {
+        org.smartregister.Context.bindtypes = new ArrayList<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.Context.bindtypes' due to existing import 'org.smartregister.Context'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5414138)